### PR TITLE
an-CX tests P0.

### PR DESCRIPTION
**Pull request by nraghavendran**
it includes the 3 tests from the an-CX test plan which are missing readme files

### CODE REVIEW

**Review by sezhang2 — DISMISSED** [review]

File: feature/gribi/otg_tests/static_lsp/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+Validate static lsp functionality.
+
+## Procedure
+
+*  Create topology ATE1–DUT1-ATE2
+*  Enable MPLS forwarding and create egress static LSP to pop the label and forward to ATE2:
+*  Match incoming label (1000001)
+*  Set IP next-hop
+*  Set egress interface
+*  Set the action to pop label (implicit_null)
```
Comment (sachendras) [inline feature/gribi/otg_tests/static_lsp/README.md:14]:
```suggestion
*  Set the action to pop label
```

File: feature/gribi/otg_tests/static_lsp/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Procedure
+
+*  Create topology ATE1–DUT1-ATE2
+*  Enable MPLS forwarding and create egress static LSP to pop the label and forward to ATE2:
+*  Match incoming label (1000001)
+*  Set IP next-hop
+*  Set egress interface
+*  Set the action to pop label
+*  Start 2 traffic flows with specified MPLS tags IPv4-MPLS[1000002]-MPLS[1000001]
```
Comment (sachendras) [inline feature/gribi/otg_tests/static_lsp/README.md:15]: I see only flow one flow w/ 2xlabels. Is that what you meant? So I think the packet will look something like below

<Ethernet><MPLS Label: 1000001><MPLS Label: 1000295><IP><TCP/UDP><DATA>

and the POP action is expected to pop `MPLS Label: 1000001`?

Comment (sachendras) [reply feature/gribi/otg_tests/static_lsp/README.md:15]: oh! so I see the following. May be the steps need to be updated

Start 2 traffic flows with specified MPLS tags
IPv4-MPLS[1000001]
IPv4-MPLS[1000002]-MPLS[1000001]

File: feature/staticroute/tests/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Summary
+
+Validate static route functionality in Management network-instance (VRF).
+
+## Procedure
+
+
+*  Configure DUT with Management VRF and ATE1, ATE2 interfaces configured within this VRF
+*  Configure IPv4 and IPv6 default routes within Management VRF pointing to ATE2 interface
```
Comment (sachendras) [inline feature/staticroute/tests/README.md:11]: Did we miss to mention what port in the DUT should be part of the VRF MGMT? There should be an interface in the same subnet as ATE2?

File: feature/gribi/otg_tests/static_lsp/README.md (hunk 2 above)
Comment (nraghavendran) [reply feature/gribi/otg_tests/static_lsp/README.md:15]: line 16 clarifies that traffic shud be received on dest ATE port with the outer label is removed.

File: feature/staticroute/tests/README.md (hunk 3 above)
Comment (nraghavendran) [reply feature/staticroute/tests/README.md:11]: line 10, specifies that the interfaces facing ate1 and ate2 from the dut, should be under the mgmt vrf.

**Review by sezhang2 — APPROVED** [review]

**Pull request merged**